bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -77,7 +77,7 @@ <h2 class="addendum-subtitle">{{ ftl('manifesto-pledge-for-a-healthy-internet')
       include_cta=True,
     ) %}
       <p>
-        <a class="mzp-c-button button-manifesto js-manifesto-share" target="_blank" rel="noopener noreferrer" href="{{ twitter_link('custom', ftl('manifesto-i-support-the-vision-of')) }}" data-link-type="social share" data-link-text="X (formerly Twitter)" aria-label="{{ ftl('manifesto-share-on-x-aria-label') }}">
+        <a class="mzp-c-button button-manifesto js-manifesto-share" target="_blank" rel="noopener noreferrer" href="{{ twitter_link('custom', ftl('manifesto-i-support-the-vision-of')) }}" data-link-text="X (formerly Twitter)" aria-label="{{ ftl('manifesto-share-on-x-aria-label') }}">
```

Comment:
Note: This would not be how I'd do this if it was a new link, but changing it now would mean we lose data continuity in GA4.